meta-xt-prod-devel-rcar-control/recipes-guests/domu/files/domu-set-root

```diff
@@ -57,7 +57,7 @@ STORAGE_PART="${BOOT_STORAGE}${PART_PREFIX}"
 
 if [ "$MODE" == "enable_virtio" ]; then
     echo "Mangling QEMU configuration: setting up block device"
-    sed -i "s/\[VIRTIO_DRIVE_CONFIGURATION\]/'backend=DomD, type=virtio,device, transport=pci, bdf=0000:00:05.0, grant_usage=1, backend_type=qemu'/g" $DOMU_CFG_FILE
+    sed -i "s/\[VIRTIO_DRIVE_CONFIGURATION\]/'backend=DomD, type=virtio,device, transport=pci, bdf=0000:00:05.0, grant_usage=1, backend_type=qemu',/g" $DOMU_CFG_FILE
```

Comment:
This already was bad, now it got even worse. Let's assume that [VIRTIO_DRIVE_CONFIGURATION] is always the last entry, or that comma is added in the config file.

Reply:
ok, this change to be dropped based on the assumption that [VIRTIO_DRIVE_CONFIGURATION] is always the last entry